http-clients/src/main/java/com/palantir/remoting3/clients/ClientConfigurations.java

```diff
@@ -87,7 +87,7 @@ public static ClientConfiguration of(
                 .enableGcmCipherSuites(DEFAULT_ENABLE_GCM_CIPHERS)
                 .proxy(ProxySelector.getDefault())
                 .proxyCredentials(Optional.empty())
-                .maxNumRetries(uris.size())
+                .maxNumRetries(DEFAULT_MAX_NUM_RETRIES)
```

Comment:
@samrogerson fysa

Reply:
uhm, I thought we had merged such a change already?

Reply:
yes, but we did it in only one of the two places (see above in this class)